assets/src/js/admin/reports/app/pages/overview-page/index.js

```diff
@@ -2,231 +2,185 @@
 const { __ } = wp.i18n;
 
 const OverviewPage = () => {
-    return (
-        <Grid>
-			<Card title={__('Donations vs Income', 'give')} width={12}>
+	return (
+		<Grid>
+			<Card title={ __( 'Donations vs Income', 'give' ) } width={ 12 }>
 				<RESTChart
-                    type='line'
-                    aspectRatio={0.4}
-                    endpoint='donations-vs-income'
-                    showLegend={false}
-                />
-            </Card>
-            <Card width={3}>
-                <MiniChart
-                    title='Mini Doughnut'
-                    type='doughnut'
-                    data={{
-                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
-                        datasets: [
-                            {
-                                label: 'Donations',
-                                data: [400.00, 532.00, 333.00, 72.56, 300.00, 422.22]
-                            }
-                        ]
-                    }}
-                />
-            </Card>
-            <Card width={3}>
-                <MiniChart
-                    title='Mini Line'
-                    type='line'
-                    data={{
-                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
-                        datasets: [
-                            {
-                                label: 'Donations',
-                                data: [400.00, 532.00, 333.00, 72.56, 390.23, 350.00]
-                            }
-                        ]
-                    }}
-                />
-            </Card>
-            <Card width={3}>
-                <MiniChart
-                    title='Mini Pie'
-                    type='pie'
-                    data={{
-                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
-                        datasets: [
-                            {
-                                label: 'Donations',
-                                data: [400.00, 532.00, 333.00, 72.56, 300.00, 450.30]
-                            }
-                        ]
-                    }}
-                />
-            </Card>
-            <Card width={3}>
-                <MiniChart
-                    title='Mini Bar'
-                    type='bar'
-                    data={{
-                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
-                        datasets: [
-                            {
-                                label: 'Donations',
-                                data: [400.00, 532.00, 333.00, 72.56, 300.00, 450.00]
-                            }
-                        ]
-                    }}
-                />
-            </Card>
-            <Card title={__('Payment Methods', 'give')} width={4}>
+					type="line"
+					aspectRatio={ 0.4 }
+					endpoint="donations-vs-income"
+					showLegend={ false }
+				/>
+			</Card>
+			<Card width={ 3 }>
+				<MiniChart
+					title="Mini Doughnut"
```

Comment:
@henryholtgeerts are you not localizing this text intentionally?

Reply:
Titles have been localized, ref: https://github.com/impress-org/givewp/pull/4429/commits/8e7b360fc2179a37eece0bdca9eb736a9153909f